axes/utils.py

```diff
@@ -69,10 +69,12 @@ def get_client_ip(request):
     return getattr(request, client_ip_attribute)
 
 
-def get_client_username(request):
+def get_client_username(request, credentials=None):
     if settings.AXES_USERNAME_CALLABLE:
-        return settings.AXES_USERNAME_CALLABLE(request)
-    return request.POST.get(settings.AXES_USERNAME_FORM_FIELD, None)
+        return settings.AXES_USERNAME_CALLABLE(request, credentials)
```

Comment:
changelog need to indicate that adding this extra param will be a breaking change and require people to update their existing AXES_USERNAME_CALLABLE  funcs

Reply:
Definitely.  
Should I change that or will one of you adjust the changelog when you release the next version with this fix included?